### PR TITLE
Migrate payment application to Micronaut

### PR DESCRIPTION
**Pull request by graemerocher**

### CODE REVIEW

File: src/payment/src/main/resources/application-oraclecloud.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    enabled: true
+    export:
+      prometheus:
+        enabled: true
+        descriptions: true
+        step: PT1M
+      oraclecloud:
+        enabled: false
+        namespace: ${ORACLECLOUD_METRICS_NAMESPACE:micronaut-mushop}
+        resourceGroup: ${ORACLECLOUD_METRICS_RESOURCEGROUP:carts}
```
Comment (pgressa) [inline src/payment/src/main/resources/application-oraclecloud.yml:20]: use `payment` instead of `carts`

File: src/payment/build.gradle [hunk 2]
```diff
@@ -0,0 +1,54 @@
+plugins {
+    id("com.github.johnrengelman.shadow") version "6.1.0"
+    id("io.micronaut.application") version "1.4.2"
+}
+
+version = "0.1"
+group = "payment"
```
Comment (pgressa) [inline src/payment/build.gradle:7]: we could agree on common group name for these service. I have used `group = "micronaut.mushop"` where with `name="carts"` it's more "real-world" scenario. But I don't mind using anything else, just for the sake of integrity ...

Comment (pgressa) [reply src/payment/build.gradle:7]: same applies to package tree .. `payment` vs `mushop.payment` ?

Comment (graemerocher) [reply src/payment/build.gradle:7]: Seems reasonable to use `micronaut.mushop`

**Pull request merged**